### PR TITLE
mxpy contract verify fixes

### PR DESCRIPTION
**Pull request by popenta**
These are the fixes that needed to be done after a discussion with @andreibancioiu , @tanghel and @florincoz .
I've tested the fixes on my local machine, but feel free to test them on your own machine.

### CODE REVIEW

File: multiversx_sdk_cli/cli_contracts.py [hunk 1]
```diff
@@ -131,14 +131,8 @@ def setup_parser(args: List[str], subparsers: Any) -> Any:
...
 
-    group = sub.add_mutually_exclusive_group(required=True)
-    group.add_argument(
-        "--project",
-        default=os.getcwd(),
-        help="🗀 the project directory (default: current directory)",
-    )
-    group.add_argument(
-        "--packaged-src", help="JSON file containing the source code of the contract"
+    sub.add_argument(
```
Comment (andreibancioiu) [inline multiversx_sdk_cli/cli_contracts.py:134]: :+1: we require `packaged-src` now.

File: multiversx_sdk_cli/cli_contracts.py [hunk 2]
```diff
@@ -147,7 +141,7 @@ def setup_parser(args: List[str], subparsers: Any) -> Any:
         required=True,
         help="the url of the service that validates the contract",
     )
-    sub.add_argument("--docker-image", required=True, help="the docker image used for the build (i.e. multiversx/sdk-rust-contract-builder:v4.0.0)")
+    sub.add_argument("--docker-image", required=True, help="the docker image used for the build (i.e. multiversx/sdk-rust-contract-builder:next)")
```
Comment (andreibancioiu) [inline multiversx_sdk_cli/cli_contracts.py:144]: Let's remove the default, and only decide on a default value one week from now?

**Review by andreibancioiu — APPROVED** [review]

**Review by andreibancioiu — APPROVED** [review]

**Review by claudiu725 — APPROVED** [review]

File: multiversx_sdk_cli/contract_verification.py [hunk 3]
```diff
@@ -80,6 +77,38 @@ def trigger_contract_verification(
...
+    status = response.get("status", "")
+    if status:
+        logger.info(f"Task status: {status}")
+
+        if status == "error":
+            dump_out_json(response)
+        else:
+            dump_out_json(response)
+    else:
+        task_id = response.get("taskId", "")
```
Comment (tanghel) [inline multiversx_sdk_cli/contract_verification.py:93]: an error can be also of other causes, so taskId should be checked here, and if it does not exist, dump the error in a standard way

Comment (popenta) [reply multiversx_sdk_cli/contract_verification.py:93]: will do!

**Pull request merged**